ios/RNSNotifications.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#pragma mark - NSNotificationCenter notification names
+
+#define RNSSafeAreaDidChange @"RNSSafeAreaDidChange"
```

Comment:
I think this header should live inside `safe-area` subdirectory. Maybe name it in a bit less general way, e.g. `RNSSafeAreaViewDefinitions` or `RNSSafeAreaViewNotifications`